converter/output/rita/streaming/dates/rita_dates.go

```diff
@@ -11,29 +11,26 @@ import (
 	"github.com/activecm/ipfix-rita/converter/logging"
```

Comment:
The streaming session writer is similar to the batch one in that, largely, the changes revolve around switching AutoFlushCollections to rita.DB objects.